src/components/navbar/Navbar.js

```diff
@@ -1,22 +1,21 @@
 import React from 'react';
-import { translate } from 'react-i18next';
-import 'i18n/i18n';
 import styles from './navbar.scss';
+import { Trans, withI18n } from '@lingui/react';
 
-const Navbar = ({ t }) => (
+const Navbar = () => (
   <nav className={styles.navigationBar}>
     <div className={styles.leftNavLinks}>
       <a href="#" className={styles.navLinks}>
-        {t('About')}
+        <Trans id="About" />
```

Comment:
for the id, let's use `[pageNameOrComponent].[uniqueKey]`, for example: `about.title`

```
<Trans id="about.title" >About</Trans>
```